app/src/lib/dispatcher/app-shell.ts

```diff
@@ -18,8 +18,9 @@ export interface IAppShell {
   readonly openExternal: (path: string) => void
   readonly openItem: (path: string) => boolean
   readonly getEditors: (path: string) => Promise<IEditorLauncher[]>
-  readonly setEditors:  (ext: string, info: IEditorInfo) => void
+  readonly setEditors:  (ext: string, info: IEditorInfo[]) => void
```

Comment:
It specifies that you can’t mutate the array, which is a good thing to verify if you don’t actually need to mutate it.